### DISCUSSION TITLE
Is there any way to give an user ALL permissions including admin?

### DISCUSSION THREAD

**Discussion by Monoverde888** [requarks/wiki#2747] [Help / Questions] [answered]
**Question**

Is there any way to give an user ALL permissions including admin?

**Host Info (please complete the following information):**
OS: Heroku
Wiki.js version: 2.5.170
Database engine: heroku postgres? (i dont know the version)

**Comment by Openanonwriter** [comment]
Could you please describe your issue more in depth. Admin should come with all the permissions. 
Create a group with a set of fallowing permissions as you see fit then add what ever user you want to the group. 

Groups> New Group> Create permissions> Add user you want to have permissions.

**Comment by Monoverde888** [comment]
I want to create an user and give him the same permissions as the admin user that is created on setup but i cant because manage:system permission is greyed out.
![greyed out](https://user-images.githubusercontent.com/43417883/100280262-a48f4100-2f46-11eb-8ec1-6b8eb50b9abf.png)

**Comment by NGPixel** [comment, marked as answer]
Just add that user to the root administrator group? Why do you want to create a second group with the same access rights?
